util/tracing/detect/detect.go

```diff
@@ -21,7 +21,10 @@ import (
 	"go.opentelemetry.io/otel/trace/noop"
 )
 
-type ExporterDetector func() (sdktrace.SpanExporter, sdkmetric.Exporter, error)
+type ExporterDetector interface {
+	DetectSpanExporter() (sdktrace.SpanExporter, error)
```

Comment:
`DetectTraceExporter` and for the rest ?

Reply:
Done.